app/src/widgets/souvenirlist.cpp

```diff
@@ -190,9 +190,9 @@ void SouvenirList::removeItem(IDs id)
 
         if(widget != nullptr && id == widget->getIDs())
         {
-            emit itemQtyChanged(id, widget->getIDs().second);
             m_IDQtys.erase(widget->getIDs());
             QListWidget::takeItem(row);
+            emit itemQtyChanged(id, -widget->getIDs().second);
```

Comment:
Wuh oh, remove all the changes in this file

Reply:
oh my bad, had to test it on this branch to make sure it worked

Reply:
oh actually no need to make a separate branch for it, these changes are how it's supposed to be